---
int16 sensorsAmount
float32 minRange
float32 maxRange
